src/main/resources/messages_ru.properties

```diff
@@ -0,0 +1,57 @@
+invalid.id=Идентификатор не может быть отрицательным
+invalid.not-empty=${validatedValue} значение не может быть пустым
+invalid.not-null=${validatedValue} значение не может быть null
+invalid.size=${validatedValue} значение должно содержать от {min} до {max} символов
+invalid.max-size=${validatedValue} значение не может превышать {max} символов
+invalid.email=Пожалуйста, введите корректный адрес электронной почты (Пример: username@example.com)
+invalid.username=Пожалуйста, введите корректное имя пользователя, не используйте пробелы
+invalid.past-or-present.date=${validatedValue} дата не может быть будущей
+invalid.password=Ваш пароль должен состоять от 8 до 30 символов и должен содержать как минимум: 1 символ верхнего регистра (A-Z), 1 символ нижнего регистра (a-z), 1 цифру (0-9)
+invalid.username.size=Имя пользователя должно состоять от 2 до 50 символов, не используйте пробелы
+invalid.username-or-password=Пожалуйста, введите корректный email/login или пароль
+invalid.phone.number=Телефонный номер должен содержать код страны без букв  (Пример: +380501234567 or +1(234)567-89-10)
+invalid.first-or-last.name=${validatedValue} значение должно содержать от 2 до 50 символов (допускаются символы: ['] и [-])
+username.duplicate=Данное имя пользователя уже используется. Пожалуйста, придумайте другое имя
+email.duplicate=Данный электронная почта уже используется. Пожалуйста, введите другой адрес или авторизируйтесь
+different.passwords=Вы ввели два различных пароля. Пожалуйста, повторите попытку
+user.not-registered=Пользователь не зарегистрирован. Некоторые поля некорректны
+
+invalid.token=Вы должны быть аутентифицированы
+token.not.start.with.bearer=JWT токен не начинается с Bearer префикса
+token.signature.not.valid=JWT подпись не соответствует локально вычисленной. JWT надежность не может быть подтверждена
+token.expired=JWT истек в
```

Comment:
Looks like something is missing)

Reply:
😊 no, there's not. I've just translated those messages, default message is "JWT expired at"